dpgen/data/tools/hcp.py

```diff
@@ -24,3 +24,12 @@ def poscar_unit (latt) :
     ret += "%.16f %.16f %.16f\n" % (1./3, 1./3, 1./2)
     return ret
 
+def STRU_unit(latt):
+
+    box = gen_box() * latt/np.sqrt(2)
+    stru_data = {}
+    stru_data['cells'] = box
+    stru_data['atom_numbs'] = [2]
+    stru_data['coords'] = np.matmul(np.array([[0.0, 0.0, 0.0], [1./3, 1./3, 1./2]]), box)
+    stru_data['types'] = [0, 0]
+    return stru_data
```

Comment:
Why not using dpdata for converting from POSCAR to STRU? You do not need to write support for abacus for each crystal type.

Reply:
> 

😯  Sorry I did not realize it when I was writing the code. I will remove the STRU_unit codes and use the dpdata conversion code.

Reply:
I've removed the STRU_unit functions and replaced them with poscar_unit and dpdata conversion.